### DISCUSSION TITLE
How to test/use saved mid checkpoints?

### DISCUSSION THREAD

**Discussion by zuencap** [TheLastBen/fast-stable-diffusion#350] [Q&A] [answered]
I used Save_Checkpoint_Every parameter so it created multiple models.

Now how can I test those models and see if they are any good? And if they are good how can I convert them to ckpt file?

Also a feature request: It would be super useful if some sample images were created during saving using the current state of the model and put together grid view of all of these images.

**Comment by TheLastBen** [comment, marked as answer]
The saved checkpoints are already in ckpt format in your gdrive, you can simply copy the path of the model and use it in the test the model cell.
as for the preview pictures, I wouldn't recommend using diffusers for that as it doesn't handle complex prompts with weights and negatives, it's best to use the A1111 cell for testing

**Reply by zuencap** [reply]
Thanks, I just noticed ckpt files in my gdrive. Would it be possible to put them under the Fast-Dreambooth folder? For tidiness :)

**Reply by TheLastBen** [reply]
I was planning on that, will change it shortly
